Jenkinsfile

```
@@ -8,7 +8,7 @@ pipeline {
 		ansiColor('xterm')
 	}
 	parameters {
-		string(name: 'CORE_VERSION', defaultValue: '4.0.1')
+		string(name: 'CORE_VERSION', defaultValue: '4.0.2')
```

Using `4.0.6` (latest version) should also be fine here but recommending `4.0.5` so that the node doesn't shutdown automatically anytime as an unintended behaviour.
```suggestion
		string(name: 'CORE_VERSION', defaultValue: '4.0.5')
```

---

Done.